### DISCUSSION TITLE
iOS publishing failed to Firebase App Distribution

### DISCUSSION THREAD

**Discussion by kterto** [codemagic-ci-cd/codemagic-docs#2510] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I configured app distribution authenticating via service account, and publishing android apk works fine, but when I build and try to distribute IOs, I'm facing an unexpected error. 
**Publishing failed :| Failed to publish /Users/builder/clone/build/ios/ipa/Viva Saúde.ipa to Firebase App Distribution: i uploading binary... Error: failed to upload release. Viva Saúde.ipa is not a legal HTTP header value .**

### Expected results

IPA can be published to Firebase app distribution.

### Actual results

Publishing failed :| Failed to publish /Users/builder/clone/build/ios/ipa/Viva Saúde.ipa to Firebase App Distribution: i uploading binary... Error: failed to upload release. Viva Saúde.ipa is not a legal HTTP header value .

**Comment by himesh-cm** [comment]
Hey @kterto, it is because the name `Saúde` and `ú` is a special character, and so Firebase complains about the legal HTTP header value while uploading the binary to the Firebase app distribution. I am unsure if you intend to use `Saúde` as the final name; it should work fine while uploading to the app store, but for Firebase app distribution, maybe please consider renaming it or manually uploading it from the firebase dashboard.

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
